src/frontend/components/Chat/SharedChatComponents/InputDisplayNameOverlay/InputDisplayNameIncorrectAlert/index.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
The commit message is not a french title 💄(frontend) add inputDisplayNameOverlay 
=> add component to display an overlay

Reply:
commit title changed : [5291ab143c0d93f5d640c686a3fe66ef8057e7c7](https://github.com/openfun/marsha/pull/1276/commits/5291ab143c0d93f5d640c686a3fe66ef8057e7c7)